graspologic/inference/latent_position_test.py

```diff
@@ -1,217 +0,0 @@
-        and second input graph, respectively. The latent positions of each sample graph
-        are used independently to sample random dot product graphs, so two null
-        distributions are generated
-
-    sample_T_statistic_ : float
-        The observed difference between the embedded positions of the two input graphs
-        after an alignment (the type of alignment depends on ``test_case``)
-
-    p_value_1_, p_value_2_ : float
-        The p value estimated from the null distributions from sample 1 and sample 2.
-
-    p_value_ : float
-        The overall p value from the test; this is the max of ``p_value_1_`` and ``p_value_2_``
-
-    See also
-    --------
-    graspologic.embed.AdjacencySpectralEmbed
-    graspologic.embed.OmnibusEmbed
-    graspologic.embed.selectSVD
-
-    References
-    ----------
-    .. [1] Tang, M., A. Athreya, D. Sussman, V. Lyzinski, Y. Park, Priebe, C.E.
-       "A Semiparametric Two-Sample Hypothesis Testing Problem for Random Graphs"
-       Journal of Computational and Graphical Statistics, Vol. 26(2), 2017
-    """
-
-    def __init__(
-        self, embedding="ase", n_components=None, n_bootstraps=500, test_case="rotation"
-    ):
-        if type(embedding) is not str:
-            raise TypeError("embedding must be str")
-        if type(n_bootstraps) is not int:
-            raise TypeError()
-        if type(test_case) is not str:
-            raise TypeError()
-        if n_bootstraps < 1:
-            raise ValueError(
-                "{} is invalid number of bootstraps, must be greater than 1".format(
-                    n_bootstraps
-                )
-            )
-        if embedding not in ["ase", "omnibus"]:
-            raise ValueError("{} is not a valid embedding method.".format(embedding))
-        if test_case not in ["rotation", "scalar-rotation", "diagonal-rotation"]:
-            raise ValueError(
-                "test_case must be one of 'rotation', 'scalar-rotation',"
-                + "'diagonal-rotation'"
-            )
-
-        super().__init__(n_components=n_components)
-
-        self.embedding = embedding
-        self.n_bootstraps = n_bootstraps
-        self.test_case = test_case
-        # paper uses these always, but could be kwargs eventually. need to test
-        self.rescale = False
-        self.loops = False
-
-    def _bootstrap(self, X_hat):
-        t_bootstrap = np.zeros(self.n_bootstraps)
-        for i in range(self.n_bootstraps):
-            A1_simulated = rdpg(X_hat, rescale=self.rescale, loops=self.loops)
-            A2_simulated = rdpg(X_hat, rescale=self.rescale, loops=self.loops)
-            X1_hat_simulated, X2_hat_simulated = self._embed(
-                A1_simulated, A2_simulated, check_lcc=False
-            )
-            t_bootstrap[i] = self._difference_norm(X1_hat_simulated, X2_hat_simulated)
-        return t_bootstrap
-
-    def _difference_norm(self, X1, X2):
```

Comment:
@kareef928 were you using a different version of the starting code? your lpt doesn't have some changes reflected in this.

Reply:
Just made a commit that made sure I'm using the most recent version of the starting code, not sure how my lpt wasn't using this version